recipes/pyopengl/post-link.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Test for OpenGL Driver on Linux
+if [ `uname` == Linux ]; then
+    $PYTHON -c "import OpenGL.GL"
```

Comment:
I had that earlier, but then it errored, because pyopengl was not installed for that Python